radio/src/gui/9X/lcd.cpp

```diff
@@ -457,9 +457,9 @@ void lcd_outdezNAtt(coord_t x, coord_t y, lcdint_t val, LcdFlags flags, uint8_t
       }
       else if (smlsize) {
```

Comment:
I had not manually tested INVERS so the Google test helped in that sense... Glad I could learn how to use GTest. Nice work guys!